snow/consensus/snowball/tree_test.go

```diff
@@ -541,7 +486,7 @@ func TestSnowballResetChild(t *testing.T) {
 	{
 		expected := `SB(Preference = 0, PreferenceStrength[0] = 1, PreferenceStrength[1] = 0, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 0
     SB(Preference = 0, PreferenceStrength[0] = 1, PreferenceStrength[1] = 0, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 1
-        SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = true)) Bits = [2, 256)
+        SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = false)) Bits = [2, 256)
```

Comment:
Virtuous instance should finalize at confidence of 2 instead of 1 after choosing a single beta value of 2.